Source/WebDriver/Session.cpp

```diff
@@ -3211,13 +3212,22 @@ void Session::doLogEntryAdded(RefPtr<JSON::Object>&& message)
     // TODO Get the current stacktrace for assert, error, trace, and warn messages
     // https://bugs.webkit.org/show_bug.cgi?id=282979
 
+    auto messageType = params->getString("type"_s);
+
     auto entry = JSON::Object::create();
-    entry->setString("type"_s, "console"_s);
+    entry->setString("type"_s, messageType);
     entry->setString("level"_s, level);
     entry->setString("text"_s, messageText);
     entry->setValue("timestamp"_s, *timestampValue);
-    entry->setString("method"_s, method);
-    entry->setArray("args"_s, args);
+
+    if (messageType == "console"_s) {
+        entry->setString("method"_s, method);
+        entry->setArray("args"_s, args);
```

Comment:
can we move the code that create `args` to inside this `if` so it doesnt have to happen for _every_ message?

or will `args` be used by other `messageType` too?

Reply:
Good catch. `args` is just used in `console` messages. I've moved it inside the `if` block and will merge after one last EWS run. Thanks for the review.